e2e-tests/ukef/cypress/e2e/journeys/portal/deal-cancellation/BSS-EWCS/deal-cancellation-maker-unable-issue-facility.spec.js

```diff
@@ -0,0 +1,86 @@
+import portalPages from '../../../../../../../portal/cypress/e2e/pages';
+import MOCK_USERS from '../../../../../../../e2e-fixtures/portal-users.fixture';
+import { generateAinUnissuedDealWithDates } from '../../test-data/AIN-deal-unissued-facilities/dealReadyToSubmit';
+import generateMinUnissuedDealWithDates from '../../test-data/MIN-deal-unissued-facilities/dealReadyToSubmit';
+import { TFM_URL, PIM_USER_1 } from '../../../../../../../e2e-fixtures';
+
+import { yesterday, tomorrow } from '../../../../../../../e2e-fixtures/dateConstants';
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+context('Deal cancellation', () => {
+  const ainUnissuedDeals = Array(2).fill(generateAinUnissuedDealWithDates());
+  const minUnissuedDeals = Array(2).fill(generateMinUnissuedDealWithDates());
```

Comment:
yes it is a draft deal. I gave this name because this deal contains unissued facilities.
`dtfs2/e2e-tests/ukef/cypress/e2e/journeys/portal/test-data/AIN-deal-unissued-facilities/dealThatJustNeedsDates.json` in this file line 101 mockFacilities.